tests/bluetooth/host/keys/mocks/keys_help_utils.c

```diff
@@ -23,15 +23,20 @@ void fill_key_pool_by_id_addr(const struct id_addr_pair src[], int size, struct
 	bt_addr_le_t *addr;
 	struct id_addr_pair const *params_vector;
 
-	zassert_true(check_key_pool_is_empty(),
-		     "List isn't empty, make sure to run this test just after a fresh start");
+	if (!check_key_pool_is_empty()) {
```

Comment:
I'm thinking instead of having something with user interaction (printing to terminal), you could just return a negative error code (of your choosing in errno.h) and assert in the caller that it returns 0.

Reply:
> I'm thinking instead of having something with user interaction (printing to terminal), you could just return a negative error code (of your choosing in errno.h) and assert in the caller that it returns 0. edit: I mean, you can leave the printout, thats cool for debugging, but also add a return value.

Ok, it's a simple change